### ISSUE TITLE
Broken link to M2O2C2: Multivariable Calculus

### ISSUE THREAD

**Issue by dmitrijskass** [kisonecat/ximera#103]
Hello,

The course "M2O2C2: Multivariable Calculus" - the link is broken. Both from the Coursera site and from http://ximera.osu.edu/, it says "502 Bad Gateway nginx/1.0.15". 

:(

**Comment by StevenGubkin** [comment]
Yes, M2O2C2 is currently broken due to a rewrite of the Ximera backend.  We have to reimplement matrices and python, basically.  This should occur in the next several months.  Sorry for this.

**Comment by kisonecat** [comment]
I am hopeful that we'll get M2O2C2 up.

**Comment by nisithsingh** [comment]
@kisonecat : Can we get a sense of when M2O2C2 will be up? Also, I love your teaching style.
